plugins/check_load.c

```diff
@@ -211,9 +206,26 @@ main (int argc, char **argv)
 		else if(la[i] > wload[i]) result = STATE_WARNING;
 	}
 
+	double scaled_la[3] = { 0.0, 0.0, 0.0 };
+	if (take_into_account_cpus == true && (numcpus = GET_NUMBER_OF_CPUS()) > 0) {
+			scaled_la[0] = la[0] / numcpus;
+			scaled_la[1] = la[1] / numcpus;
+			scaled_la[2] = la[2] / numcpus;
+
+			char *tmp = NULL;
+			xasprintf(&tmp, _("load average: %.2f, %.2f, %.2f"), scaled_la[0], scaled_la[1], scaled_la[2]);
+			xasprintf(&status_line, "scaled %s - %s", tmp, status_line);
+	}
+
 	printf("LOAD %s - %s|", state_text(result), status_line);
-	for(i = 0; i < 3; i++)
-		printf("load%d=%.3f;%.3f;%.3f;0; ", nums[i], la[i], wload[i], cload[i]);
+	for(i = 0; i < 3; i++) {
+		if (scaled_la[i] != 0.0) {
```

Comment:
i don't think it's a good idea to make printing the performance values depend on the actual value. Either you want it or not, but don't magically hide some values if they are zero.

Reply:
This is completely true, this was the "poor man's version" of testing whether the scaled values are actually in usage or not.
Solved this another way now.